packages/core-plugin-api/src/extensions/extensions.tsx

```diff
@@ -101,6 +103,15 @@ export function createRoutableExtension<
                 }
                 throw error;
               }
+
+              // This event, never exposed to end-users of the analytics API,
+              // helps inform which extension metadata gets associated with a
+              // navigation event when the route navigated to is a gathered
+              // mountpoint.
+              useEffect(() => {
+                analytics.captureEvent('_routable-extension-rendered', '');
```

Comment:
Feeling it's worth defining an internal constant for `'_routable-extension-rendered'`